cnv/cnv_install/preparing-cluster-for-cnv.adoc

```diff
@@ -0,0 +1,14 @@
+[id="preparing-cluster-for-cnv"]
+= Preparing your {product-title} cluster for {ProductName}
+include::modules/cnv-document-attributes.adoc[]
+:context: preparing-cluster-for-cnv
+toc::[]
+
+{ProductName} 2.0 works with {product-title} by default, however the following 
+installation configurations are recommended:
+
+* The {product-title} is installed on 
```

Comment:
assuming product-title is openshift, i would add the word 'cluster'
The {product-title} cluster is installed on...